### ISSUE TITLE
Updated link styling in the body of eScholarship pages

### ISSUE THREAD

**Issue by lschiff** [eScholarship/jschol#999]
Link styling should be changed as follows:

1. Links in the body of eScholarship pages should be underlined; leave the breadcrumb, global header and footer areas unlinked.  ([Accessibility issue item # 81](https://docs.google.com/spreadsheets/d/1kIMiW09tBtk5d3kJjzUMSCo1tEWPn7bUc0BYAJes9Hw/edit?gid=0#gid=0))
2.  Author names in the "Recent Articles author list" and "Related Items author list" should not be linked. ([Accessibility issue items # 82 & #83](https://docs.google.com/spreadsheets/d/1kIMiW09tBtk5d3kJjzUMSCo1tEWPn7bUc0BYAJes9Hw/edit?gid=0#gid=0))

Mockups in this [Figma file](https://www.figma.com/design/LoDeC5KjNg4eQXetfAY6QK/eScholarship-link-design?node-id=60-2&t=PLuRt4LHXNQDoP9r-1).
